source/item_pool.cpp

```diff
@@ -484,6 +483,19 @@ static Item GetJunkItem() {
   return JunkPoolItems[idx];
 }
 
+int PlaceJunkInExcludedLocation(ItemLocation * il) {
```

Comment:
The return value of this doesn't seem to be used anywhere, was that intended?

Reply:
It should just be void. I probably intended to use the return value for something while programming but didn't end up using it and then forgot about it.